pyqg/qg_diagnostics.py

```diff
@@ -29,7 +30,8 @@ def _initialize_model_diagnostics(self):
                         self.a, self._calc_parameterization_contribution()))
                 ),
             units='m^2 s^-3',
-            dims=('l','k')
+            dims=('l','k'),
+            skip_comparison=True,
```

Comment:
I'm not happy about adding more attributes to the diagnostics dictionaries, but I think we can address that as we refactor diagnostics to first-class objects.